modules/ethereumNode.js

```diff
@@ -350,14 +361,39 @@ class EthereumNode extends EventEmitter {
 
                 let args;
 
-                // START TESTNET
-                if (network == 'test') {
-                    args = (nodeType === 'geth')
-                        ? ['--testnet', '--fast', '--ipcpath', Settings.rpcIpcPath]
-                        : ['--morden', '--unsafe-transactions'];
-                }
-                // START MAINNET
-                else {
+                switch (network) {
+                // STARTS ROPSTEN
+                case 'test':
```

Comment:
There are some reasons:
- backwards compatibility, yes
- Geth refers to the networks as `--testnet` and `--rinkeby`, so we're actually being consistent with their flags
- forward compatibility: that'll work even if the testnet flag changes to a different chain, so we're not being specific to Ropsten in the internals, only when it's displayed.